### DISCUSSION TITLE
🐞 grpc connection closed when using podman + wsl2

### DISCUSSION THREAD

**Discussion by marcosnils** [dagger/dagger#12255] [Help]
### What is the issue?

When using podman 5.1.2 + wsl2 and trying to do `dagger init` on an already existing codebase, Dagger fails with a grpc client connection closed error. 

Discord thread: https://discord.com/channels/707636530424053791/1267572025791090759

note: I have the suspicion that this might be related to some podman client grpc timeout issue given that on every test that I saw failing, the error was always failing after around 30s the pipeline started consistently. I've also found this issue in the podman repo https://github.com/containers/podman/issues/20379#issuecomment-2222685142 which seems relevant as it's also a long running operation.

### Dagger version

v0.12.3

### Steps to reproduce

1. Have a wsl2 + podman >= 5.1.2 setup
2. Perform a `dagger init --sdk $sdk` on an already existing project

### Log output

 ! failed to export: rpc error: code = Canceled desc = grpc: the client connection is closing
Error: failed to generate code: input: blob.diff.export resolve: failed to export: rpc error: code = Canceled desc = grpc: the client connection is closing

**Comment by jedevc** [comment]
Note: we should fix this, but also we should have provisioning tests for podman - currently this isn't tested :cry: 

We should also have for windows, which I'm working on, but it's a pain :cry:
